Software/model/model.py

```diff
@@ -0,0 +1,19 @@
+from Software.model import place
+
+
+class Model(object):
+
+    def __init__(self, numberOfPlaces):
+        self.numberOfPlaces = numberOfPlaces;
+        self.list_of_labels_to_display_ticket_number = [];
+        self.list_of_labels_to_display_place_number = [];
+        self.list_of_labels_to_display_in_queue = [];
```

Comment:
Please remove the `;` at the end of these lines, and also at `self.initPlaces();` some lines below

Reply:
fixed